DISCOVER/09_participant_selection.md

```diff
@@ -81,8 +82,8 @@
       - If you recognize a submission author and you&#39;re on the review committee, you should not grade the submission—recuse yourself from reviewing that submission.
       - Optionally, the review team can de-anonymize after the review process to ensure that the final selection is representative of the diverse grouping that is desired.
   - **Double Open Review**
-    - Reviewers know who the proposal authors are and proposal authors know who the reviewers are
-    - [Sample guidelines for reviewers in double open review systems](https://scipy2017.scipy.org/ehome/220975/532468/)
+    <!--- Reviewers know who the proposal authors are and proposal authors know who the reviewers are
```

Comment:
hmm not sure why this would be the right place to start the comment. The point is still valid even if the comment is not.